tmt/steps/prepare/feature/epel-enable.yaml

```diff
@@ -14,11 +15,13 @@
             name: yum-utils
             state: present
 
-        - name: Enable repos of 'epel'
-          shell: yum-config-manager --enable epel epel-debuginfo epel-source
+        - name: Enable EPEL repos
+          ansible.builtin.command: yum-config-manager --enable epel epel-debuginfo epel-source
+          register: output
+          changed_when: output.rc != 0
 
-    # RHEL N, where N >= 8
-    - when: ansible_distribution == "RedHat" and ansible_distribution_major_version|int >= 8
+    - name: Enable EPEL and EPEL-Next repos on RHEL 8+
+      when: ansible_distribution == "RedHat" and ansible_distribution_major_version | int >= 8
       block:
         - name: Install package 'epel-release'
           ansible.builtin.yum:
```

Comment:
@psss not related to this PR, but wouldn't it make sense to use `ansible.builtin.dnf`? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html

Reply:
Ack, yeah, plus there are a couple of other simplifications (like installing two packages in a single task). @martinhoyer, would you be interested in creating a separate pull request?